db/dbtypes/types.go

```diff
@@ -258,12 +245,12 @@ func AgendaStatusFromStr(status string) AgendaStatusType {
 	}
 }
 
-// AddrTxnType enumerates the different transaction types as displayed by the
-// address page.
-type AddrTxnType int
+// AddrTxnViewType enumerates the different address transaction view types as
+// displayed by the address page.
+type AddrTxnViewType int
```

Comment:
The previous name implied it was the *transaction's* type (e.g. vote, revocation, etc), but it's better described as the page view.